build.csx

```diff
@@ -183,7 +186,7 @@ targets.Add(
         var version = GetVersion();
         foreach (var nuspec in nuspecs)
         {
-            Cmd(nuget, $"pack {nuspec} -Version {version} -OutputDirectory {outputDirectory} -NoPackageAnalysis");
+            Cmd("nuget", $"pack {nuspec} -Version {version} -OutputDirectory {outputDirectory} -NoPackageAnalysis");

```

Comment:
it's a mistake. I don't know what I did here... It might have happened when I tried to use `dotnet restore` instead of nuget.